config.py

```diff
@@ -467,22 +467,39 @@
 
 #  `event_id`` : dict
```

Comment:
```suggestion
#  `event_id``: dict
```
no?

Reply:
I believe in numpydoc one is supposed to have the space, and that's also how it has been handled in the rest of `config.py`. I think we should keep the space for sake of consistency for now :)

Reply:
I meant : `` -> `

Reply:
Oh. it's actually supposed to be two backticks on either side. Fixed. sorry about that.